scriptmodules/emulators/mupen64plus.sh

```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/env bash
 
 # This file is part of The RetroPie Project
-# 
+#
 # The RetroPie Project is the legal property of its developers, whose names are
 # too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
-# 
+#
```

Comment:
All the copyright headers contain this whitespace. We should clean all if changing it so search/replace is easier in future. I would leave it as is in this file for now.